cleancat/base.py

```diff
@@ -344,7 +349,8 @@ def is_valid(self):
             return True
 
     def serialize(self, value):
-        return self.schema_class(data=value).serialize()
+        if value is not None:
```

Comment:
This one is an interesting case because I would expect it to be `None` (just like you implemented) if the embedded object isn't set. But I'd expect the parent class `Dict` to be an empty dict if not set since it's unstructured.

Reply:
Agreed again. See the latest tests to see how embedded objects (and embedded references) are serialized as None, but Lists/SortedSets/Dicts default to `[]`/`{}`.